creator-node/src/snapbackSM/peerSetManager.js

```diff
@@ -20,7 +21,7 @@ const MAX_NUMBER_SECONDS_PRIMARY_REMAINS_UNHEALTHY = config.get(
   'maxNumberSecondsPrimaryRemainsUnhealthy'
 )
 
-const DEFAULT_AXIOS_TIMEOUT_MS = 5000 // 5s
+const DEFAULT_AXIOS_TIMEOUT_MS = 60_000 // 60s
```

Comment:
bit concerned about setting this "default axios timeout" to 60s, realistically its only used in one place so we should define it inside `getNodeUsers()` so other calls don't use it?

wdyt

Reply:
I think I prefer it being in `getNodeUsers()` too since `asyncRetry` already has a default timeout.

Reply:
great